pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -802,6 +805,13 @@ func (m *migrationMonitor) startMonitor() {
 	m.lastProgressUpdate = m.start
 
 	logger := log.Log.Object(vmi)
+	defer func() {
```

Comment:
Can you explain the need to zero out all the fields here?

Reply:
discussed.